docs/serving/autoscaling/autoscaling-metrics.md

```diff
@@ -32,6 +32,8 @@ For more information about KPA and HPA, see the documentation on [Supported Auto
         metadata:
           annotations:
             autoscaling.knative.dev/metric: "concurrency"
+            autoscaling.knative.dev/target-utilization-percentage: "70"
+            #autoscaling.knative.dev/target-utilization-percentage sepcifies a percentage value which when reached autoscaling is performed 
```

Comment:
rather than making these types of comments in the yaml (which isn't super easy to read), it might be better to put this info after the yaml block

Reply:
Helo @psschwei 
Many thanks for your feedback. I have performed the suggested changes. Kindly review and thanks again!